src/main/java/ch/njol/skript/expressions/ExprExperience.java

```diff
@@ -94,18 +96,20 @@ public Class<?>[] acceptChange(final ChangeMode mode) {
 	}
 	
 	@Override
-	public void change(final Event e, final @Nullable Object[] delta, final ChangeMode mode) {
-		double d;
+	public void change(Event e, @Nullable Object[] delta, ChangeMode mode) {
+		int d;
```

Comment:
if this variable is getting changed to an integer, the field name should be changed. Rather than d for double, we can just make it something like experience.